### ISSUE TITLE
[D] Is getting different, cost-equivalent, equipment like a monitor, an option?

### ISSUE THREAD

**Issue by bzz** [src-d/guide#25]
Great to to see [available_hardware.md](https://github.com/src-d/guide/blob/master/general/available_hardware.md) published - awesome list of laptops!

I have a question - does anybody think that getting a different, cost-equivalent, equipment like a monitor, rather than the [listed ones](https://github.com/src-d/guide/blob/master/general/available_hardware.md#monitor) could possibly be an option? 

I.e a there is almost new rotating Dell P2715Q on sale in one of the mobile market apps in Madrid for ~400e right now (which is 700e new), and it’s very similar to the ones that are already been used in office, but its by far better then ~370e ones from the list.

I understand the benefits of having a single standard for the company, but at the same time having some flexibility could go a long way in productivity and convenience (in case it does not bring many downsides i.e for items that do not have a very long life span). 

What do you think?

**Comment by eiso** [comment]
My personal opinion is that for laptops I understand the standardisation, since they are quite expensive and if somebody leaves they can be upgraded and handed over easily. Also makes it practically a lot easier with people setting up their system on day 1 (I know that the XPS and Lenovo options are there because they have good Linux compatibility).

For peripherals like monitors, I wouldn't see any reason why not to allow someone to buy something of equivalent price. It's an extra effort and hassle but if someone is willing to make that effort, why not?

This is just my 2 cents, in the end the final decision on this is @mcuadros responsibility. 

p.s. I am linking this issue in [available_hardware.md](https://github.com/src-d/guide/blob/master/general/available_hardware.md) so that in the future people can understand the reasoning.

**Comment by eiso** [comment]
I hadn't thought before about the warranties & invoice implication. Unfortunately that rules out 2nd hand hardware unless we buy it from a company and they can issue an invoice for it and provide the warranty documentation. 

@jorgeschnura can we add links to the checkout of the laptops or maybe their value so that people know what equivalent amount means. 

@bzz if you find any other options for 4k screens within that price range, definitely suggest them. 

So to summarise, *if anything is of equivalent monetary value, then we can buy it instead of what is on the list as long as it is new or in the rare case of it being second hand but an invoice and warranty can be provided*.

I will update the [available_hardware.md](https://github.com/src-d/guide/blob/master/general/available_hardware.md) to reflect this discussion.

**Comment by jorgeschnura** [comment]
I disagree with this. We offer several very nice options already that I think represent a good variety of "tastes".
The problem with making it free to choose is that then the company ends up having a huge stock of random hardware that can't be re-used in most cases. This is what has happened before, where we would allow everyone to choose their own hardware and we ended up with a big pile of random laptops that now we have to sell at a discount and buy new laptops at full price, which makes the company lose money.
We can discuss changing some of the options presented if you find better alternatives though or if you think a concreted taste is not represented here.

**Comment by eiso** [comment]
@mcuadros @jorgeschnura and I spoke about this offline (no new opinions were raised then already mentioned above) and came to a decision. 

Let's make the [available_hardware.md](https://github.com/src-d/guide/blob/master/general/available_hardware.md) list open to suggestions via pull requests. If you think anything is missing that can make your life better, suggest it, you need the LGTM from @mcuadros for it be merged and then @estherrgarcia can help with ordering and keeping track of the hardware. 

I'll close this issue for now but if anyone in the future wants to reopen it, feel free to do so. Also as we grow more successful as a company, it is likely we will update this list with more high value items, right now this is also constrained by budget.
